components/list/list-item.js

```diff
@@ -195,30 +203,55 @@ class ListItem extends LinkMixin(RtlMixin(LitElement)) {
 		fastdom.measure(() => this.resizedCallback(this.offsetWidth));
 	}
 
+	get checked() {
+		return this._checked;
+	}
+
+	set checked(value) {
+		const oldVal = this.checked;
+		this._checked = value;
+		this.requestUpdate('checked', oldVal).then(() => {
+			const checkBox = this.shadowRoot.querySelector('.d2l-list-item-checkbox');
+			checkBox && (checkBox.checked = value);
+
+			if (typeof this.ref === 'undefined') {
+				this.ref = ListItem.uniqueRef();
```

Comment:
I don't really understand what's going on with all this ref code. If you want to ensure that the ref/key is set, couldn't you just set a default value to `getUniqueId()` in the constructor?

Reply:
Yeah, I can do that. 

The reason I didn't do getUniqueId is because it was technically an UniqueId so there wasn't a reason to bring that global in. However it gets the job done.

- [ ] Just use getUniqueId